pkg/apis/core/types_shoot.go

```diff
@@ -627,6 +627,8 @@ type OpenIDConnectClientAuthentication struct {
 type AdmissionPlugin struct {
 	// Name is the name of the plugin.
 	Name string
+	// Disabled describes whether this plugin should be disabled in the kube-apiserver
```

Comment:
Nit:

```suggestion
	// Disabled specifies whether this plugin should be disabled.
```

Same for other types files.